src/Sylius/Bundle/ApiBundle/Serializer/ProductSerializer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Sylius\Component\Core\Model\ProductInterface;
+use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductSerializer implements ContextAwareNormalizerInterface
```

Comment:
This solution fixes us on taking only 3 reviews per page, we should use index limits in this case. It is enough to use build in function:
![Uploading image.png…]()